db/test/unit/mutations/submit_application_test.sql

```diff
@@ -89,6 +98,34 @@ select results_eq(
   'Increases the ccbc number when submitting applications'
 );
 
+select throws_like(
+  $$
+    select ccbc_public.submit_application(5)
+  $$,
+  'The application cannot be submitted as the submission field submission_date is null or empty'
```

Comment:
```suggestion
  'The application cannot be submitted as the submission field submission_date is null or empty',
  'Throws an exception if submission_date is empty'
```
Adding test descriptions will improve logging if the tests fail

Reply:
I don't see how this is any more helpful or descriptive than what is already there, which already shows in the logs when a test fails.

Reply:
Fair, I assumed the log was more cryptic without a drescription